h/static/scripts/util/url-util.js

```diff
@@ -16,7 +16,7 @@ function replaceURLParams(url, params) {
       var value = params[param];
```

Comment:
Possibly. I'll note that this function is currently used to generate API URLs for resources using the annotator store API description returned by `/api` which currently uses `:foo` rather than `{foo}` placeholders.